pkg/ocicni/ocicni.go

```diff
@@ -54,9 +54,8 @@ func (plugin *cniNetworkPlugin) monitorNetDir() {
 				}
 
 				if err = plugin.syncNetworkConfig(); err == nil {
-					logrus.Debugf("CNI asynchronous setting succeeded")
-					close(plugin.monitorNetDirChan)
-					return
+					logrus.Infof("CNI asynchronous setting succeeded")
```

Comment:
Will this logrus.Infof put too much info into the log?  Will the log just be filled with this message not adding much value?

Reply:
I think this would only be logged if the CNI Changes?

Reply:
Yes, it will only log on CNI changes.